### DISCUSSION TITLE
Custom Connectors? Connecting to substrate chains

### DISCUSSION THREAD

**Discussion by niklasp** [rainbow-me/rainbowkit#2150] [Q&A] [answered]
👋 Is the code extendible to write custom connectors to e.g. Polkadot. Some wallets (e.g. Talisman) supports substrate (polkadot sdk) based chains and i wonder how to go about writing a custom connector to a non eth based chain.

**Comment by Adebesin-Cell** [comment]
You mean without ``window.ethereum``?

**Comment by DanielSinclair** [comment, marked as answer]
@niklasp Only Ethereum and the broader EVM ecosystem are supported today by Wagmi, Viem, and RainbowKit. We do support Polkadot wallets that also support EVM.
